worlds/shivers/Constants.py

```diff
@@ -1,17 +1,19 @@
-import os
 import json
+import os
 import pkgutil
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
 
 def load_data_file(*args) -> dict:
     fname = os.path.join("data", *args)
     return json.loads(pkgutil.get_data(__name__, fname).decode())
 
-location_id_offset: int = 27000
 
+location_id_offset: int = 27000
 location_info = load_data_file("locations.json")
-location_name_to_id = {name: location_id_offset + index \
-	for index, name in enumerate(location_info["all_locations"])}
-
+location_name_to_id = {name: location_id_offset + index for index, name in enumerate(location_info["all_locations"])}
 exclusion_info = load_data_file("excluded_locations.json")
-
 region_info = load_data_file("regions.json")
+years_since_sep_30_1980 = relativedelta(datetime.now(), datetime.fromisoformat("1980-09-30")).years
```

Comment:
Is there a reason this uses relativedelta instead of something simple like:
```suggestion
years_since_sep_30_1980 = datetime.now().years - 1980
```

Reply:
Or maybe something like:
```py
datetime.now().year - 1980 + (datetime.now().month < 10)
```

Reply:
Sep 30 1980 is a very specific date in the game. It's the date that the mystery officially occurs, and the game takes place in 1995, sometime in October or November (The game was released in November) though not specified.

![image](https://github.com/user-attachments/assets/af87b354-ff38-4dad-9f4e-eb73afe992e6)
![image](https://github.com/user-attachments/assets/1af94c4e-85ac-483c-ac0e-e9315e8a9594)

So the victory item is basically an in joke where you've finally solved the mystery... X years later from that date. Relative delta just for it's accuracy honestly. (Though sadly the item name won't change if the game starts before Sep 30th but ends after Sep 30th).

Reply:
Removed dateutil